### ISSUE TITLE
Rerunning PydocstringFormat deletes file contents if docstrings already present

### ISSUE THREAD

**Issue by jrwrigh** [heavenshell/vim-pydocstring#104]
If you run `:PydocstringFormat` on a file that already has docstrings, it simply deletes the contents of the file. 

``` py
#!/usr/bin/env python

def formGrid(x, y=None):
    z = x + y

    return z
```

Run `:PydocstringFormat`:

``` py
#!/usr/bin/env python

def formGrid(x, y=None):
    """formGrid.   
      
    Parameters    
    ----------
    x :
        x
    y :
        y
    """
    z = x + y

    return z
```

Then running it again deletes the contents of the file.

**Comment by heavenshell** [comment]
1st
```python
#!/usr/bin/env python

def formGrid(x, y=None):
    z = x + y

    return z
```
`:PydocstringFormat:`

outputs
```python
#!/usr/bin/env python

def formGrid(x, y=None):
    """formGrid.   
      
    Parameters    
    ----------
    x :
        x
    y :
        y
    """
    z = x + y

    return z
```

2nd
`:PydocstringFormat:`

outputs
```python
#!/usr/bin/env python

def formGrid(x, y=None):
    """formGrid.   
      
    Parameters    
    ----------
    x :
        x
    y :
        y
    """
    z = x + y

    return z
```

all expected behaviors.

What do you mean `deletes the contents of the file.`
What is your expected behavior?

**Comment by jrwrigh** [comment]
> What do you mean deletes the contents of the file.

This:
![pydocFormatBug20200710](https://user-images.githubusercontent.com/20801821/87216719-8ad6bf80-c2ff-11ea-854d-bdfbf5a7209d.gif)
<sup>Note I did change formatting styles in this gif. I've been able to recreate it on multiple styles</sub>

> What is your expected behavior?

Not that ^^^^

**Comment by heavenshell** [comment]
@jrwrigh Thank you for upload screencapture 👍 

humm, but I could not reporduce.
![Kapture 2020-07-11 at 18 34 18](https://user-images.githubusercontent.com/56591/87221240-69c8aa00-c3a5-11ea-9d93-a2b55e020569.gif)

Is doq and pydocstring is latest version?
Did this bug raised at minimamu vimrc?

```vim
set nocompatible
filetype off
set rtp=$VIMRUNTIME

" Add pydocstring folder into rtp
set rtp+=/path/to/vim-pydocstring
filetype plugin indent on
```

**Comment by edkirk30** [comment]
I'm also getting this behaviour on NVIM v0.4.3 including with the minimal vimrc supplied.

Installed latest version via vim-plug

**Comment by jrwrigh** [comment]
doq: 0.6.1
vim-pydocstring: `master`
Neovim: v0.4.3

I can also confirm that it does happen with the minimal rc provided.

**Comment by heavenshell** [comment]
@jrwrigh @edkirk30 
Fixed and release as 2.3.2
Thank you for reporting.
